AirCasting/ABConnector/BluetoothManager.swift

```diff
@@ -8,7 +8,23 @@
 import Foundation
 import CoreBluetooth
 
-class BluetoothManager: NSObject, ObservableObject {
+protocol BluetoothCommunicator {
```

Comment:
Hmmm... I think this is already on develop. Could fix your branch so that we would have only your changes in this PR?